package.json

```diff
@@ -174,7 +174,7 @@
     "typescript": "^5.8.3",
     "unist-util-visit": "^5.0.0",
     "util": "^0.12.5",
-    "vite": "^7.0.6",
+    "vite": "catalog:",
     "vitest": "catalog:",
```

Comment:
I'm confused on this one. I would have expected the opposite move. To what I understand catalog is for "dependencies", where we can't have one package at the root's package.json. In the case of devDependencies, we can, so catalog only add indirections, so should be removed?

Reply:
> Ah, you mean that we don't care about having different versions of dev dependencies?

I imagine that if a dev dependency is shared between packages, we can have it set at the root package.json, in which case, we can't have multiple version.

Reply:
> we can have it set at the root package.json

That was the case before this PR, but the `build` script in `test/regressions` didn't see it, so I had to declare it there too: https://github.com/mui/mui-x/pull/18970#discussion_r2240629193